Chrysalis-Papilio/Soen490-Papilio-Website
# PAPILIO-95-task Adding statistics graph

## General 
Each companies have many statistics that could help them with the management of their business on Papilio. Those are some of the statistics that a company use:

- Activity view (number of people that view their activity)
- Activity registration (number of people that registered to an activity)

Those are important statistics that needs to be made available to the company. The main page of the dashboard is the most suitable place to place them since it allows employee to easily and quickly grasp the situation.

_Issue number_: Papilio-95

## Task description
- Add Bar Graph component
- Add graph to Dashboard home page 
- Add toggle between activity
- Little bit of style
- Add corresponding test

## Manual Test

Make sure that you have the appropriate variables in the backend .env file. (Contact me if you don't)

1. start the backend
`npm run build && npm start`
2. start the website
`npm start`
3. On the main page navigate to the login page
4. Insert the admin account information
5. Once logged in, you should see the graph that were created
6. You can toggle the different activities and see the graph change with the activity

## Testing
To run all the unit tests, run the following code: `npm test`
To make sure that the tests have reach the appropriate level of coverage, run the following code: `npm run coverage`
